tsdb/mapper.go

```diff
@@ -16,8 +16,9 @@ import (
 // aggregate output Time is ignored, and actual Time-Value pairs are contained soley
 // within the Value field.
 type MapperValue struct {
-	Time  int64       `json:"time,omitempty"`  // Ignored for aggregate output.
-	Value interface{} `json:"value,omitempty"` // For aggregate, contains interval time multiple values.
+	Time  int64             `json:"time,omitempty"`  // Ignored for aggregate output.
+	Value interface{}       `json:"value,omitempty"` // For aggregate, contains interval time multiple values.
+	Tags  map[string]string `json:"tags,omitempty"`  // Meta tags for results
```

Comment:
That's on the user. Generally if you want the data, you're best doing a `GROUP BY *` instead of pulling it in the raw values.

I think this is the right way to do it. And in the case of aggregates, retransmitting the tags isn't a big issue since you're talking about many fewer map output results anyway